pom.xml

```diff
@@ -1921,11 +1921,6 @@
         <version>${antlr4.version}</version>
       </dependency>
       <dependency>
-        <groupId>jline</groupId>
```

Comment:
This caused a problem in Scala 2.12, but, doesn't seem like we have any direct dependency on jline anyway